server/tests-py/test_compression.py

```diff
@@ -63,27 +63,28 @@ def test_gzip_compression_v1_query(self, hge_ctx):
         resp = self._make_post(hge_ctx, url, q, self.gzip_header)
         self._assert_gzip(resp, exp_resp)
 
-    def test_brotli_compression_graphql(self, hge_ctx):
-        url, q, exp_resp = self._get_config(self.dir() + '/graphql_query.yaml')
-        resp = self._make_post(hge_ctx, url, q, self.brotli_header)
-        self._assert_brotli(resp, exp_resp)
+    # def test_brotli_compression_graphql(self, hge_ctx):
+    #     url, q, exp_resp = self._get_config(self.dir() + '/graphql_query.yaml')
+    #     resp = self._make_post(hge_ctx, url, q, self.brotli_header)
+    #     self._assert_brotli(resp, exp_resp)
 
-    def test_brotli_compression_v1_query(self, hge_ctx):
-        url, q, exp_resp = self._get_config(self.dir() + '/v1_query.yaml')
-        resp = self._make_post(hge_ctx, url, q, self.brotli_header)
-        self._assert_brotli(resp, exp_resp)
+    # def test_brotli_compression_v1_query(self, hge_ctx):
+    #     url, q, exp_resp = self._get_config(self.dir() + '/v1_query.yaml')
+    #     resp = self._make_post(hge_ctx, url, q, self.brotli_header)
+    #     self._assert_brotli(resp, exp_resp)
```

Comment:
Could you also remove these commented-out tests? (Sorry for being nitpicky about this, since it’s kind of a minor thing, but it’s easy to fix, so it seems worth doing.)